README.md

```diff
@@ -22,6 +23,11 @@ following environment variables must/can be set:
   batch job, so this should be set to the job ID.
 - `GRANULE_ID`: ID of the HLS granule from which to produce the associated HLS
   VI granule files.
+- `LPDAAC_PROTECTED_BUCKET_NAME` (_optional_): Name of the bucket containing
+  ingested HLS granule files (default: `"lp_prod_protected"`).
+- `LPDAAC_PUBLIC_BUCKET_NAME` (_optional_): Name of the bucket containing
+  HLS thumbnails (`.jpg`) (default: `"lp_prod_public"`) associated with the
+  granule files in the other bucket.
```

Comment:
The names are correct in code but should be dash separated `-` instead of `_` (`lp-prod-protected`, not `lp_prod_protected`)

Reply:
Good eyes! 👀